lib/google_contacts_api/contact.rb

```diff
@@ -1,19 +1,36 @@
 module GoogleContactsApi
+
   # Represents a single contact.
-  # Methods we could implement:
-  # :categories, (:content again), :links, (:title again), :email
-  # :extended_properties, :deleted, :im, :name,
-  # :organizations, :phone_numbers, :structured_postal_addresses, :where
   class Contact < GoogleContactsApi::Result
-    # Returns the array of links, as link is an array for Hashie.
-    def links
-      self["link"].map { |l| l.href }
+
+    attr_reader :json, :original_contact
+
+    #this selects the first address in list we have
+    def primary_address
```

Comment:
I thought the primary address was the address where `primary: true`? Not necessarily  the first address.